Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/ParametersSectionTranslator.swift

```diff
@@ -20,15 +20,27 @@ struct ParametersSectionTranslator: RenderSectionTranslator {
         translateSectionToVariantCollection(
             documentationDataVariants: symbol.parametersSectionVariants
         ) { _, parameters in
-            guard !parameters.parameters.isEmpty else { return nil }
+            guard !(parameters.parameters.isEmpty || parameters.parameters.count < 1) else {
+                return ParameterRenderSection(name: "", content: []) as? RenderSection
+            }
             
             return ParametersRenderSection(
                 parameters: parameters.parameters
                     .map { parameter in
                         let parameterContent = renderNodeTranslator.visitMarkupContainer(
                             MarkupContainer(parameter.contents)
                         ) as! [RenderBlockContent]
-                        return ParameterRenderSection(name: parameter.name, content: parameterContent)
+                        
+                        guard !parameterContent.isEmpty && parameterContent.count != 0 else {
```

Comment:
Same here  - why check `count` in addition to `isEmpty`?

Reply:
Same as above — must have forgotten to clean build, resulting in a strange build state / build errors.